docs/debug/index.md

```diff
@@ -157,6 +160,8 @@ def calculate_sum_and_averages(numbers: list) -> list:
     return list_so_far
 ```
 
+The file is opened in append mode, so if a file already exists, the new table will be added at the end. If the file does not currently exist, then a new file will be created.
```

Comment:
This sentence is unnecessary, as this information is already provided on Line 137